### ISSUE TITLE
Running on Ubuntu 20.04

### ISSUE THREAD

**Issue by eN0Rm** [emrahcom/emrah-buster-base#1]
How hard would it be to make this work on Ubuntu 20.04 too?

**Comment by emrahcom** [comment]
I don't think it will be hard to adapt to Ubuntu but it's difficult for me to maintain it because I don't use Ubuntu so much. I test this work on many different environment (AWS, DigitalOcean, Virtualbox, real machine etc) with different components
